test/models/concerns/podcast_subscribe_links_test.rb

```diff
@@ -0,0 +1,25 @@
+require "test_helper"
+
+class PodcastSubscribeLinksTest < ActiveSupport::TestCase
+  let(:podcast) { create(:podcast) }
+  let(:podcast_2) { create(:podcast) }
+  let(:apple_link) { SubscribeLink.create(platform: "apple", podcast: podcast, external_id: "12345") }
+
+  before do
+    assert apple_link
+  end
+
+  describe ".build_subscribe_links_json" do
```

Comment:
i wasn't sure if these tests would be helpful or if there is something different i should be testing for with these methods. like formatting?

Reply:
I think these are about right.  In this case, the "formatting" done in the method is pretty trivial.  So testing just the business logic (returns nil when absent) makes sense.

Reply:
Nitpick: the standard when writing tests is to use `.class_method_name` or `#instance_method_name`.  Since these are instance methods, they should start with a `#`.  (Little counter-intuitive yeah, but that's the standard!)